pyproject.toml

```
@@ -66,8 +66,7 @@ select = [
     "FURB177", # Prefer `Path.cwd()` over `Path().resolve()` for current-directory lookups
     "FURB187", # Use of assignment of `reversed` on list `{name}`
     # PYI: only enable rules that have autofixes and that we always want to fix (even manually),
```

In a separate PR (or this one if you prefer), this could be changed to
```suggestion
    # PYI: only enable rules for which we never noqa their flake8-pyi plugin equivalent,
```

Even better imo is to instead select all and disable the few codes where this happens.